common/protob/messages-cardano.proto

```diff
@@ -138,13 +145,58 @@ message CardanoSignTx {
         optional uint64 amount = 3;                                     // amount to spend
         optional CardanoAddressParametersType address_parameters = 4;   // parameters used to derive the address
     }
+
+    /**
+     * Stakepool owner parameters
+     */
+    message CardanoPoolOwnerType {
+        optional bytes staking_key_hash = 1;
+        repeated uint32 staking_key_path = 2;
+    }
+
+    /**
+     * Stakepool relay parameters
+     */
+    message CardanoPoolRelayParametersType {
+        required CardanoPoolRelayType type = 1;
+        optional bytes ipv4_address = 2;
+        optional bytes ipv6_address = 3;
+        optional string host_name = 4; 
+        optional uint32 port = 5;
+    }
+
     /**
+     * Stakepool metadata parameters
+     */
+    message CardanoPoolMetadataType {
+        required string url = 1;
+        required bytes hash = 2;
+    }
+
+    /**
+     * Stakepool parameters
+     */
+    message CardanoPoolParametersType {
+        required bytes pool = 1;
+        required bytes vrf_key_hash = 2;
+        required uint64 pledge = 3;
+        required uint64 cost = 4;
+        required uint64 margin_numerator = 5;
+        required uint64 margin_denominator = 6;
+        required bytes reward_account_key_hash = 7;
+        repeated CardanoPoolOwnerType owners = 8;
+        repeated CardanoPoolRelayParametersType relays = 9;
+        optional CardanoPoolMetadataType metadata = 10;
+    }
+
+        /**
      * Structure representing cardano transaction certificate
      */
-    message CardanoTxCertificateType {
-        optional CardanoCertificateType type = 1;   // certificate type
-        repeated uint32 path = 2;                   // BIP-32 path to derive (staking) key
-        optional bytes pool = 3;                    // pool hash
+     message CardanoTxCertificateType {
+        optional CardanoCertificateType type = 1;                 // certificate type
+        repeated uint32 path = 2;                                 // BIP-32 path to derive (staking) key
+        optional bytes pool = 3;                                  // pool hash
```

Comment:
renaming this would introduce inconsitency with the already existing js api which calls this parameter "pool" arguably introducing as much confusion. But it makes sense to me to rename like this the "pool" parameter in CardanoPoolParametersType as it is kind of weird having a "pool" parameter inside such structure.